components/openindiana/nwam-manager/nwam-manager.p5m

```diff
@@ -27,7 +28,7 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 file path=etc/gconf/schemas/nwam-manager.schemas
 file path=etc/xdg/autostart/nwam-manager.desktop
-file path=usr/bin/nwam-manager-properties mode=0555
+file path=usr/bin/nwam-manager-properties
 file path=usr/lib/nwam-manager mode=0555
```

Comment:
Why doesn't nwam-manager-properties needed to be executable?

Reply:
> Why doesn't nwam-manager-properties needed to be executable?

Because nobody should execute such strange command.  No, just kidding :-).

The reason is that we do not need explicit mode for _normal_ binaries because https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/transforms/defaults#L113